content/blog/2022/2022-06-01-blog-minions-new.md

```diff
@@ -0,0 +1,63 @@
++++
+author = "Zala Gruden"
+date = "2022-06-02"
+draft = false
+title = "Teaching fourth-graders data science"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-06-02-hands-up.jpg"
+frontPageImage = "/blog_img/2022/2022-06-02-hands-up.jpg"
+blog = ["Elementary school", "Tides Foundation", "project", "classification"]
+shortExcerpt = "We visited our local elementary school and explained the kids how classification works."
+longExcerpt = "We visited fourthgraders in our local elementary school and taught them about clustering and data collection."
++++
+
+{{< window-screenshot src="/blog_img/2022/2022-06-02-working-together.jpg" >}} 
+
+What if we told you that it is possible to teach data mining to kids in elementary school in an enjoyable way? You’d say, 
+
+"Impossible!" and 
+"That's too much to ask of a kid!". 
+
+We say, 
+"Just watch; it can be done in a fun and informative way that will help these kids learn fundamental logical thinking patterns!" 
+
+
+A week before our Tuesday visit, the kids from Alojzij Šuštar elementary school in Ljubljana took a trip to the Ljubljana ZOO, paying close attention to animals and their features. They were given a form (Figure 1) into which they had to enter the data for three mammals, three birds, fish, amphibians, reptiles, insects or spiders, and invertebrates (excluding insects and spiders). For every animal, they wrote the species and several features: hairy, "slimy", aquatic, has fingers, has ears, and can fly. After returning to school, the teacher entered the data into a Google Form. The children were split into 7 groups, so we had a data set with around 140 instances (some referred to the same animal, yet possibly with different data).
+
+{{< window-screenshot src="/blog_img/2022/22-05-31 form.png" >}}
+*(Figure 1) Showing the form that the kids to enter their animal data into.* 
+ 
+Remember this; we will be coming back to this story at the end! 
+
+On the day of our visit, Janez gave the fourth-graders from the ZOO a new task - Minion classification. 
```

Comment:
V internem poročilu, namenjenem nam in Googlu lahko pišemo o minionih, v blogu pa zaradi copyrighta raje ne bi. Ajda predlaga, da govorimo o kakšnih "characters" ali čem podobnem, odvisno od konteksta. Predlagam torej, da vse pojavitve "minion" zamenjaš s primernejšimi drugimi besedami.